packages/action-button/src/ActionButton.ts

```diff
@@ -210,7 +210,13 @@ export class ActionButton extends SizedMixin(ButtonBase, {
     protected override updated(changes: PropertyValues): void {
         super.updated(changes);
         const isButton = this.role === 'button';
-        const canBePressed = isButton && (this.selected || this.toggles);
+        const canBePressed =
+            isButton &&
+            (this.selected || this.toggles) &&
+            !(
+                this.hasAttribute('aria-haspopup') &&
+                this.hasAttribute('aria-expanded')
+            );
```

Comment:
We should definitely surface a test for this somewhere, either here in directly in the Action Menu. A comment might be nice as well.